### PR TITLE
Allow deleting a story from the story's show page

### LINKED ISSUE #240: Allow deleting a story from the story's show page
**Branch/Commit:**

Main branch

**Describe the feature:**

When I'm reading a story, I should have a button to delete the story. After confirmation it deletes the story and takes me back to the project that contains it.

**Problem:**

When you are linked to a story, if you want to delete it you have to go back to the project show page with the listing, look for the story and then click the 3 dots and then delete.

**I will abide by the [code of conduct](https://github.com/fastruby/points/blob/main/CODE_OF_CONDUCT.md)**

### PR DESCRIPTION
**Pull request by JuanVqz**
Closes #240

I will abide by de [code of conduct](https://github.com/fastruby/points/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by JuanVqz** [comment]
Peek 2022-04-06 08-35

**Comment by JuanVqz** [comment]
it seems like the failing test is a random failure, should I resolve it?
`rspec ./spec/features/stories_manage_spec.rb:87`

**Review by fbuys — APPROVED** [review]
Looks good to me! 🚀

**Comment by fbuys** [comment]
@rishijain could you please do QA on this. And if it looks good to you then please merge it in 🙏

**Comment by rishijain** [comment]
hey @JuanVqz I have QA this and everything seems to be working as per expectations.

File: app/views/stories/show.html.erb [hunk 1]
```diff
@@ -6,5 +6,6 @@
   <div class="btn-group">
     <%= link_to 'Edit', edit_project_story_path(@project), id:"back", class: "button" %>
     <%= link_to 'Back', project_path(@project), id:"back", class: "button" %>
+    <%= link_to "Delete", project_story_path(@project.id, @story), method: :delete, data: { confirm: "Are you sure?", story_id: @story.id }, class: "button red", remote: true , title: "Delete"%>
```
Comment (JuanVqz) [inline app/views/stories/show.html.erb:9]:
```suggestion
    <%= link_to "Delete", project_story_path(@project.id, @story), method: :delete, data: { confirm: "Are you sure?", story_id: @story.id }, class: "button red", remote: true , title: "Delete" %>
```

**Comment by rishijain** [comment]
Thank you for the work on this @JuanVqz :)

**Pull request merged**